src/components/navbar/book-links.cjsx

```diff
@@ -25,8 +25,6 @@ BookLinks = React.createClass
         </a>
       )
 
-    window.open(course.webview_url, 'cc-webview-link')
-
```

Comment:
Aha!  that's why I had the render returning null in booklinks.  I'd noticed the popup blocker saying windows were blocked and had put it on my to-investigate-later list.  That line was from the old method where it was using window.open in a method vs just using an `a` tag.